File: TypeSharpSample/TypeSharpModules/BusinessObjects/IOrderAssignable.ts
/// <reference path="BusinessObjects.d.ts" />

module BusinessObjects {

	export interface IOrderAssignable {
	
		/**
		* Assigns the Order to assignable
		* @param order The order to assign
		* @return true on success
		*/
		AssignOrder(order: Order): bool;
	
	}
	
}

File: TypeSharpSample/BusinessObjects/IOrderAssignable.cs
using System;
using TypeSharp;
namespace BusinessObjects
{
    [TypeSharpInterface]
    public interface IOrderAssignable
    {
        /// <summary>
        /// Assigns the Order to assignable
        /// </summary>
        /// <param name="order">The order to assign</param>
        /// <returns>true on success</returns>
        bool AssignOrder(Order order);
    }
}
